docker-compose.yml

```diff
@@ -103,6 +103,32 @@ services:
     volumes:
       - ./build/certs:/etc/certs
       - ./build/mongod.conf:/etc/mongod.conf
+  
+  # used for testing a replica set
+  mongodb_replset:
+    build:
+      context: ./build/deps
+      dockerfile: ${MONGO_DOCKERFILE:-mongo6}.Dockerfile
+    container_name: mongodb_replset
+    command: mongod --port 27017 --bind_ip_all --replSet rs0
+    ports:
+      - 47017:27017
```

Comment:
This uses the same host port `47017` as mongodb above, is this on purpose?

Reply:
Yes it is, that's the port number for [`proxy-addr`](https://github.com/FerretDB/FerretDB/blob/0b16cdbb87577facd9066185e2d269c41612a1d6/Taskfile.yml#L327). We'd never run a replica set and a standalone side by side I expect as well.